### DISCUSSION TITLE
No module named "'netbox"

### DISCUSSION THREAD

**Discussion by bahner** [netbox-community/netbox#7448] [Q&A]
After upgrade to netbox 3-ldap in docker, I get the stangest error message on the front page.
--------------------------------------------
Server Error
A module import error occurred during this request. Common causes include the following:

Missing required packages - This installation of NetBox might be missing one or more required Python packages. These packages are listed in requirements.txt and local_requirements.txt, and are normally installed as part of the installation or upgrade process. To verify installed packages, run pip freeze from the console and compare the output to the list of required packages.

WSGI service not restarted after upgrade - If this installation has recently been upgraded, check that the WSGI service (e.g. gunicorn or uWSGI) has been restarted. This ensures that the new code is running.

The complete exception is provided below:

<class 'ModuleNotFoundError'>

No module named "'netbox"

Python version: 3.9.5
NetBox version: 3.0.5

------------------------------

Obviously it looks like a typo, where a module is prefixed with a single quote. But I can find any reference to this in the code. And no error messages occur in the logs.

[netbox.txt](https://github.com/netbox-community/netbox/files/7287563/netbox.txt)

**Comment by jeremystretch** [comment]
Try enabling debug (`DEBUG=True` in `configuration.py`) to get a full stack trace. That should tell you exactly where the error lies.

**Reply by bahner** [reply]
That might've helped:

AUTHENTICATION_BACKENDS	
["'netbox.authentication.LDAPBackend'",
 'netbox.authentication.ObjectPermissionBackend']
 
 Let's see.

**Reply by bahner** [reply]
And indeed, that was the error. /me slaps forehead.

Thanks for the obvious tip.

**Reply by jeremystretch** [reply]
No worries, glad you got it sorted!
